app/src/main/AndroidManifest.xml

```diff
@@ -4,9 +4,9 @@
 
     <application
         android:name=".GovUkApplication"
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
The guidance I got from Jon and Ryan for this ticket was for the following:

- Allow backups
- Exclude db data
- Include shared prefs like iOS (as apparently iOS couldn't exclude them)

?

Reply:
I think that will lead to unexpected behaviour as the app will think the user has completed topic selection (as the flag is stored in shared prefs) but there will be no items in the database. Wouldn't it make more sense to start them with a clean slate?

Reply:
@TimGJD Had a chat with Ryan and he has agreed that it makes sense to exclude everything for now.